src/patterns.rst

```diff
@@ -1174,6 +1174,18 @@ deconstructor`.
 A :t:`binding` of a :t:`binding pattern` binds a matched :t:`value` to a
 :t:`name`.
 
+:dp:`fls_RViC5UEZPQUV`
+A :t:`binding` with :t:`binding mode` :dt:`by value` binds the matched :t:`value`
+by :t:[pass]ing` to the :t:`name`.
```

Comment:
"by passing to the name" reads strangely. Did you mean "binds the matched value to a name", "binds the matched value to a name by passing the value to the place indicated by the name", "binds the matched value to the place of a name", or something similar?